app/uk/gov/hmrc/eeittadminfrontend/controllers/SubmissionSdesController.scala

```diff
@@ -157,21 +159,34 @@ class SubmissionSdesController @Inject() (
       .fold(
         _ =>
           Redirect(
-            routes.SubmissionSdesController.sdesSubmissions(page, None, None, None)
+            routes.SubmissionSdesController.sdesSubmissions(page, None, None, None, None)
           ).pure[Future],
         {
-          case (maybeFormTemplateId, maybeStatus) =>
-            Redirect(
-              routes.SubmissionSdesController.sdesSubmissions(
-                0,
-                None,
-                maybeFormTemplateId.map(FormTemplateId(_)),
-                maybeStatus.map(NotificationStatus.fromString(_))
-              )
-            ).pure[Future]
+          case (maybeFormTemplateId, maybeStatus, maybeShowBeforeAt) =>
+            if (maybeShowBeforeAt.getOrElse(false) == true) {
```

Comment:
This condition seems to do nothing. If `maybeShowBeforeAt` is `None`, `else` branch is executed which sets `showBeforeAt` to `None`, which is exactly what would happen if thee was no `if..else` condition.